pkg/certificate/providers/certmanager/certificate_manager.go

```diff
@@ -239,6 +245,8 @@ func NewCertManager(
 	namespace string,
 	issuerRef cmmeta.ObjectReference,
 	cfg configurator.Configurator,
+	validityPeriod time.Duration,
```

Comment:
s/validityPeriod/serviceCertValidityDuration

Same comment applies to other certificate provider init.